src/operators/assignment_operator.cpp

```diff
@@ -30,7 +30,29 @@
 
 #include "../bigint.hpp"
 
-namespace libbig {
+namespace libbig
+{
+  void largeInt::operator=(largeInt x)
```

Comment:
modify the declaration of this function to recieve a `const largeInt&` instead of `largeInt`, this will avoid copying the whole string, just a bit of optimization

Reply:
and please change the formatting scheme, rest everythiing looks good to me for now in this file